ducktape/services/service.py

```diff
@@ -69,23 +68,58 @@ def __repr__(self):
                              (self.num_nodes, [n.account.hostname for n in self.nodes]))
 
     @property
+    def name(self):
+        """Human-readable identifier (almost certainly) unique within a test run."""
+        return self.__class__.__name__ + "-" + str(self.id)
```

Comment:
Right, those use cases seem valid. I was just wondering if a test-specific override could be handy, e.g. rather than getting something like `ConsoleConsumer-0` I could get something like `ConsoleConsumer-ValidateMessages`. (I don't have a good example of where we use the same service multiple times right now, but hopefully you get the idea).

In any case, no need to add the override now either, just a thought.